metaci/release/utils.py

```diff
@@ -55,9 +55,9 @@ def update_release_from_github(release, repo_api=None):
     return release
 
 
-def send_release_webhook(project_config, release):
-    if release is None or not settings.METACI_RELEASE_WEBHOOK_URL:
-        return
+def send_release_webhook(project_config, release, config_item):
+    if release is None or not settings.METACI_RELEASE_WEBHOOK_URL or not settings.GUS_BUS_ENABLED or not config_item:
```

Comment:
The point of the new setting is to give us a way to keep the existing gus-bus integration enabled (for creating the change case) but not enable the start/stop calls yet. So we should not check the new setting here. If we need a way to toggle all of the webhooks at once, we can already do that using METACI_RELEASE_WEBHOOK_URL.

Also for now we still need to send the release webhook and create a change case even if we don't have a config item. We can update it to be stricter later once we've set up the config item path for all our orgs.